src/planet-a/Active/SysPlanetA.cpp

```diff
@@ -125,6 +132,11 @@ void SysPlanetA::draw(osp::active::ACompCamera const& camera)
         auto& planet = drawGroup.get<ACompPlanet>(entity);
         auto& transform = drawGroup.get<ACompTransform>(entity);
 
+        if (planet.m_planet.get() == nullptr)
```

Comment:
it's sufficient to just do "planet.m_planet == nullptr", you don't need to call .get().